spec/requests/api/v1/users_request_spec.rb

```diff
@@ -60,9 +60,11 @@
                       email: 'james.p.sullivan@aol.com',
                       password: "gw45635yhethet5"
                     })
-      headers = {"CONTENT_TYPE" => "application/json"}
-    
-      post api_v1_users_path, headers: headers, params: JSON.generate(user: user_params)
+      # headers = {"CONTENT_TYPE" => "application/json"}
+      # post api_v1_users_path, headers: headers, params: JSON.generate(user: user_params)
+
+      #Above (line:63-64) is what was here, and below ln:67 is the new suggestion `EB`
+      post api_v1_users_path, params: user_params
```

Comment:
Could you elaborate on what exactly this difference is doing? I thought that the original syntax including the headers was necessary for the request to go through?